docs/reference/inferenceservice/getting_started.md

```diff
@@ -0,0 +1,124 @@
+     Since this model was trained under the COCO dataset, it's label maps are available [here](https://github.com/tensorflow/models/blob/master/research/object_detection/data/mscoco_label_map.pbtxt).
+
+	Our final result is something like this:
+   * ssd
+     * 0001
+       * saved_model.pb
+       * label_map.pbtxt
+       * variables
+         * variables.data-00000-of-00001
+         * variables.index
+
+3. Upload files under your object storage provider ( **Azure Storage**, **S3** or **GCS** ).
+4. Then go to **Models** > **New Model Server**.
+5. Copy and paste the YAML below, replacing &lt;namespace&gt; with your namespace
+	```yaml
+  	apiVersion: "serving.kubeflow.org/v1beta1"
+  	kind: "InferenceService"
+  	metadata:
+      namespace: <namespace>
+      name: ssd # name of model server
+  	spec:
+      transformer:
+        containers:
+        - image:  onepanel/transformer-tfod-base64:v1.0.0
+          name: kfserving-container
+          env:
+           - name: STORAGE_URI
+             value: "s3://ssd/" # since Onepanel uses minio, all cloud providers can use the s3 prefix
+           - name: model
+             value: ssd # make sure this is the same as metadata.name
+      predictor:
+        tensorflow:
+          runtimeVersion: "2.5.1"
+          storageUri: "s3://ssd/"
+	```
+6. Click **Create**
+
+7. To test the model, use this [jupyter notebook](https://github.com/onepanelio/python-sdk/blob/master/examples/deploy-and-consume-inference-api.ipynb)
+
+## Create custom model servers with Workflows
+
+You can also deploy models using workflows.
+Using the `ssd` example above, we can replace step 5 with the following
+
+1. Go to **Workflows** > **Workflow Templates** > **Create Template**
+2. Assign template name and then under manifest paste the following:
+	```yaml
+  	entrypoint: main
+  	templates:
+      - dag:
+          tasks:
+            - name: deploy
+              template: deploy
+        name: main
+      - name: deploy
+        resource:
+          successCondition: status.address.url
+          action: create
+          manifest: |
+            apiVersion: "serving.kubeflow.org/v1beta1"
+            kind: "InferenceService"
+            metadata:
+              namespace: "{{workflow.namespace}}"
+              name: "{{workflow.name}}"
+            spec:
+              transformer:
+                containers:
+                - image: onepanel/transformer-tfod-base64:v1.0.0
+                  name: kfserving-container
+                  env:
+                   - name: STORAGE_URI
+                     value: "s3://ssd"
+                   - name: model
+                     value: "{{workflow.name}}"
+              predictor:
+                tensorflow:
+                  runtimeVersion: "2.5.1"
+                  storageUri: "s3://ssd"
+	```
+3. This will generate the model servers and assign names automatically.
+4. Execute the workflow
```

Comment:
Need to add another page "Create with Python SDK" and this one can simply point to the two example notebooks we have here: https://github.com/onepanelio/python-sdk/tree/master/examples